### DISCUSSION TITLE
OpenSnitch running on Librem5!

### DISCUSSION THREAD

**Discussion by gustavo-iniguez-goya** [evilsocket/opensnitch#376] [Show and tell]
https://twitter.com/i/web/status/1370504114866511876

![image](https://user-images.githubusercontent.com/2742953/111082890-f914f700-850a-11eb-95ac-b292f7f6e7e5.png)

**Comment by themighty1** [comment]
Bravo!!!

**Comment by gustavo-iniguez-goya** [comment]
https://puri.sm/posts/snitching-on-phones-that-snitch-on-you/

**Reply by roke-julian-lockhart** [reply]
@gustavo-iniguez-goya, is the GUI adequately optimized for a low DPI vertical aspect ratio, and is it usable with touch input?

**Reply by gustavo-iniguez-goya** [reply]
hey @RokeJulianLockhart ,

No, the GUI is not specifically optimized for the mobile. Since v1.3.6 I changed some widgets/functionality for a better integratation with the desktop. For example by using dbus notifications, now we have notifications on the desktop panel even if the systray is not available (on Gnome and Phosh it's not available for example). 

Also now you can configure what columns to display on the Events tab, which should improve the user experience when viewing the events on a mobile. But I haven't tested all the latest changes on a Purism/Librem5 image (I don't own a Librem5).

On the other hand, we could easily change (or let the user change) the GUI size using qt-material: https://github.com/UN-GCPDS/qt-material#density-scale
